src/main/lib/utils.js

```diff
@@ -1,8 +1,14 @@
 import path from 'path';
 
-export function getPathToFile(fileName) {
+export function getPathToFile(fileName, disableFilePath = false) {
```

Comment:
I would suggest using a whole new function for this, rather than having a cryptic boolean flag.. What about `getURIPathToFile` and `getPathToFile`?

Changing them would be easy due to their infrequent use.